interface/main/tabs/js/tabs_view_model.js

```diff
@@ -372,6 +373,29 @@ function menuActionClick(data,evt)
 
 }
 
+function reportMenuClickData(data, evt = 'menuClick') {
+    top.restoreSession();
+    const clickEventData = {
+        action: 'reportMenuClickData',
+        eventType: evt,
+        eventLabel: data.label(),
+        eventUrl: data.url(),
+        eventTarget: data.target,
+        csrf_token_form: top.csrf_token_js,
+    };
+
+    let url = top.webroot_url + '/library/ajax/track_events.php';
+    fetch(url, {
```

Comment:
I verified this is all async non-blocking so it won't cause any performance penalties on the client side.